lab-justin/route/note-route.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const Note = require('../model/note');
+const response = require('../lib/response');
+// const Note = require('../lib/storage');
+
+// const note = notePool[req.url.query.id];
+// const note = new Note(req.body.content);
+
+var notePool = {};
+
+module.exports = function(router){
+
+  router.post('/api/note', function(req, res){
+    if (req.body){
+      const note = new Note(req.body.content);
+      notePool[note.id] = note;
+      return response(200, note)(res);
+    }
+    response(400, 'bad request')(res);
+  });
+
+  router.get('/api/note', function(req, res){
+    const note = notePool[req.url.query.id];
```

Comment:
This is dangerous! Your code will break if they make a request to this route without a query string.